platform-sdk/platform-apps/tests/MigrationTestingTool/src/main/java/com/swirlds/demo/migration/AccountCodec.java

```diff
@@ -8,26 +8,26 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.io.IOException;
 
-public class AccountVirtualMapValueCodec implements Codec<AccountVirtualMapValue> {
+public class AccountCodec implements Codec<Account> {
 
-    public static final AccountVirtualMapValueCodec INSTANCE = new AccountVirtualMapValueCodec();
+    public static final AccountCodec INSTANCE = new AccountCodec();
 
-    private static final AccountVirtualMapValue DEFAULT_VALUE = new AccountVirtualMapValue(0, 0, 0, false, 0);
+    private static final Account DEFAULT_VALUE = new Account(0, 0, 0, false, 0);
```

Comment:
It probably makes sense to introduce a no-arg constructor for `Account` and use it here